databricks_template_schema.json

```diff
@@ -316,6 +331,44 @@
           }
         ]
       }
+    },      
+    "input_docker_image": {
+      "order": 20,
+      "type": "string",
+      "description": "\nDocker image for the execution of Gitlab pipelines",
+      "default": "registry.gitlab.com/ TODO /",
```

Comment:
What is the TODO for here?

Reply:
I assume that the user creates a Docker image with all the required libraries to run the unit tests and pushes it in their own or in a public registry.
(I need to document it)

Reply:
I've updated the file
 template/{{.input_root_dir}}/docs/mlops-setup.md.tmpl
to describe the process to create and push the docker image